Sources/Models/SubscriberState.swift

```diff
@@ -9,10 +9,12 @@ public enum SubscriberState {
     status: Stripe.Subscription.Status, enterpriseAccount: EnterpriseAccount?, deactivated: Bool)
 
   public init(
-    user: User?, subscriptionAndEnterpriseAccount: (Models.Subscription, EnterpriseAccount?)?
+    user: User?,
+    subscription: Models.Subscription?,
+    enterpriseAccount: EnterpriseAccount?
   ) {
-    switch (user, subscriptionAndEnterpriseAccount) {
-    case let (.some(user), .some((subscription, enterpriseAccount))):
+    switch (user, subscription) {
+    case let (.some(user), .some(subscription)):
```

Comment:
We were using a weird optional tuple for an argument to the init due to how the middleware transformers aggregated all the data together. I've greatly simplified that and so this can now be simpler.